eclair-core/src/main/resources/reference.conf

```diff
@@ -121,6 +121,8 @@ eclair {
     sync {
       request-node-announcements = true // if true we will ask for node announcements when we receive channel ids that we don't know
       encoding-type = zlib // encoding for short_channel_ids and timestamps in query channel sync messages; other possible value is "uncompressed"
+      channel-range-chunk-size = 2500 // do not change this unless you know what you are doing
```

Comment:
What's the rationale for changing the default to from 3500 to 2500? Is it improving performances?

Reply:
This is related to:

> @sstone pretty sure this calculation was wrong, can you reproduce it?

The max value was wrong because we didn't take timestamps+checksums into account. It is somewhere around 2700.